build/netradiant/install/pkg/default_shaderlist.txt

```diff
@@ -0,0 +1 @@
+common
```

Comment:
Again, why pre-populate a build directory? Potentially other editors may need the same file, too.